src/main/java/seedu/address/logic/commands/RemoveTagCommand.java

```diff
@@ -20,13 +24,20 @@
     public static final String COMMAND_USAGE = COMMAND_WORD + " ";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Removes specified tag\n"
-            + "Parameters: TAG (must be a string)\n"
-            + "Paramaters: INDEX (must be a positive integer) TAG (must be a string)";
+            + "Parameters: RANGE (all OR INDEX(Index must be positive)) followed by TAG (must be a string)\n"
+            + "Example: removetag 30 prospective OR removetag all colleagues";
 
     public static final String MESSAGE_SUCCESS = "Tag removed";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
     public static final String MESSAGE_NOT_DELETED = "Tag not deleted";
-    public static final String MESSAGE_EXCEEDTAGNUM = "Please only type one TAG to be removed";
+    public static final String MESSAGE_EXCEEDTAGNUM = "Please type one TAG to be removed";
+    public static final String MESSAGE_TAG_NOT_FOUND = "Tag given does not exist in address book";
+    public static final String MESSAGE_TAG_NOT_FOUND_IN = "Index %s does not have the given tag.";
+
+    public static final int RANGE = 0;
+    public static final int TAG = 1;
```

Comment:
Don't think I saw 'WHOLEAB`, `RANGE` and `TAG` constants being used in `RemoveTagCommand`, maybe the three constants can be moved to `RemoveTagCommandParser` class?